MSMAll/scripts/MSMAll.sh

```diff
@@ -75,9 +75,10 @@ PARAMETERs are [ ] = optional; < > = user supplied value
    --rerun=TBW
    --reg-conf=TBW
    --reg-conf-vars=TBW
-  [--matlab-run-mode={0, 1}] defaults to 0 (Compiled MATLAB)
+  [--matlab-run-mode={0, 1, 2}] defaults to 0 (Compiled MATLAB)
```

Comment:
I thought we were going to try to default to using Interpreted MATLAB across all scripts, for consistency.